src/main/java/newbank/Configuration.java

```diff
@@ -6,4 +6,10 @@
 
     public static final String DEFAULT_CURRENCY = "GBP";
     public static final int MAX_ACCOUNTS = 5;
+
+    public static double MAX_MICROLOAN = 1000;
+    public static double ACCRUAL_RATE = 0.01;
```

Comment:
Can we convert to a safer type such as the already used BigDecimal?

Reply:
Ooh, yes. And MAX_MICROLOAN really ought to be Money, come to think of it! (I mean, 1000 *what*?!)